Exchange-Guide.md

```diff
@@ -0,0 +1,190 @@
+- `BTCD` - full node, connected to the BTC chain
+- `LTCD` - full node, connected to the LTC chain
+- `LND-BTC` - lightning network daemon for the BTC network
+- `LND-LTC` - lightning network daemon for the LTC network
+- `XUD` - Exchange Union's decentralized exchange layer
+
+## Installation 
+### Prerequisites
+Make sure you have the below programs installed:
+- [git](https://gist.github.com/derhuerst/1b15ff4652a867391f03#file-linux-)
+- [make](https://www.cyberciti.biz/faq/debian-linux-install-gnu-gcc-compiler/)
+- [node.js + npm](https://nodejs.org/en/download/package-manager/#debian-and-ubuntu-based-linux-distributions)
+- python2.7 (preinstalled on most systems, check with `python --version`)
+- killall (`sudo apt-get install psmisc`)
+
+### Installing GO
+if GO is not yet installed on your system please use the following command to install it:
+```
+sudo apt-get install golang-1.10-go
+```
+Note that golang-1.10-go puts binaries in /usr/lib/go-1.10/bin. If you want them on your PATH, you need to make that change yourself. Alternatively, you can create a softlink as follows:
+```
+sudo ln -s /usr/lib/go-1.10/bin/go /usr/local/bin/go 
+```
+
+### Repository Checkout
+From your home directory run
+```
+git clone https://github.com/ExchangeUnion/xud-simnet.git
+```
+This should create the `xud-simnet` directory under your home directory.
+
+### Setup ~/.bashrc
+To enable access to the scripts please update and source your `.bashrc`:
+```
+source ~/xud-simnet/setup.bash
+source ~/.bashrc
+```
+Please note that the `setup.bash` script set your `$GOPATH` to `~/xud-simnet/go`. All changes from `setup.bash`are temporary and only active for the current terminal session. You need to repeat the sourcing above or add things to `PATH` to make these permanent.
+
+### Verify Setup
+Before you move on lets verify that we are ready:
+```
+xud-simnet-check
+```
+
+### Installation
+Install all components with
+```
+xud-simnet-install
+```
+It could take several minutes depending on the speed of your machine & connection until you see `Ready!`.
+
+### Starting
+You can start all components with
+```
+xud-simnet-start
+```
+Since `btcd` and `ltcd` need to sync the blocks of the XUD simnet when started for the first time, it could take a minute or two until you see `Ready!`.
+
+### Payment Channels
+To setup payment channels run
+```
+xud-simnet-channels
+```
+Payment channels are used to instantly settle trades via cross chain atomic swaps with peers. It should take about 10 minutes for your payment channels to be ready. The `xud-simnet` features one minute block times and we could speed this up arbitrarily, but later on mainnet this will involve an even more significant waiting time. We decided to keep it close to reality without being to annoyingly long.
```

Comment:
Should I implement https://github.com/ExchangeUnion/xud-tests/issues/4 so that the waiting time will be non-existent?

Reply:
No, as discussed in the call we'll leave it with the 1 min block times